app/views/neighborhoods/index.html.erb

```diff
@@ -6,6 +6,6 @@
   <p><%= neighborhood.location %></p>
 <% end %>
 
-<% if current_user %>
-  <%= link_to 'New Neighborhood', new_neighborhood_path %>
+<% if !current_user.nil? && current_user.admin? %>
```

Comment:
consider a `Neighborhood.creatable_by?` class method here similar to the wavelength I've indicated above.